src/main/scala/net/ripe/rpki/publicationserver/metrics/Metrics.scala

```diff
@@ -18,6 +18,23 @@ import java.io.StringWriter
 import io.prometheus.client.exporter.common.TextFormat
 import io.prometheus.client._
 
+
+object Metrics {
```

Comment:
Can you add 1-2 lines describing what this does/what it is used for? This is not clear for me right now without a deeper investigation (especially since I thought metrics were already merged)